src/org/usfirst/frc/team2585/TeleopExecuter.java

```diff
@@ -0,0 +1,21 @@
+package org.usfirst.frc.team2585;
+
+import org.impact2585.lib2585.RunnableExecuter;
+import org.usfirst.frc.team2585.systems.Initializable;
+
+public class TeleopExecuter extends RunnableExecuter implements Initializable {
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = 1L;
```

Comment:
Eclipse can generate a serialVersionUID for you. In order to generate the serialVersionUID, delete this, click on the lightbulb and select the generate serialVersionUID option.